.github/workflows/remote-push.yml

```diff
@@ -26,3 +26,15 @@ jobs:
             Gi_per_thread: 4
             python: ${{ matrix.python }}
         secrets: inherit
+
+    # Benchmarks
+    AWS-AVX2-192G-4-A10G-96G-Benchmark:
+        uses: ./.github/workflows/nm-benchmark.yml
+        with:
+            label: aws-avx2-192G-4-a10g-96G
```

Comment:
maybe this could just a single GPU? it's just a 7b model so 4 GPUs seems like overkill

Reply:
+1

Reply:
Thanks guys.
4 GPU instance (Gi_per_thread = 4):
 - build time : 10 mins
 - bench time : 20 mins
1 GPU instance (Gi_per_thread = 12):
 - build time : 30 mins 
 - bench time : 20 mins    
 
 I am trying to see if it builds with Gi_per_thread = 8.